### DISCUSSION TITLE
Conditional Fields

### DISCUSSION THREAD

**Discussion by himanshuahuja96** [directus/directus#4533] [Feature Requests]
It would make be awesome to have conditional fields, repeater like WordPress ACF.

Here is reference link to that.
https://www.advancedcustomfields.com/resources/field-settings/

We are looking for Headless CMS for our new lms system and we are right now evaluating some of the best option. And we found directus and we liked it on first look.

Thank you for all of your hard work.

**Comment by benhaynes** [comment]
Duplicate: https://github.com/directus/directus/issues/3151
